docs/specs/code/SERVER_STATUS_ENDPOINT.md

```diff
@@ -0,0 +1,76 @@
+# Abstract
+
+Describe a new endpoint to notify server issues to Yoroi-frontend, describe the front-end changes to handle and display this information.
+
+# Motivation
+
+Currently, Yoroi-frontend has no way to notify neither server issues nor connection issues. A new endpoint notifying the server status can be implemented in the back-end and the front-end can get this information periodically to display information if a problem is reported. Also, if the endpoint doesn't respond, the front-end can assume there is a connection issue.
+
+Also, once Yoroi has the ability to notify server issues, the endpoint used by Yoroi to fetch the information can be used to create a public URL that reports the mentioned status, such as [AWS](https://status.aws.amazon.com/) and [Twitter](https://status.twitterstat.us).
+
+# Proposal
+
+## Backend
+
+Adding a new endpoint in the backend named `api/status` is proposed. This endpoint has the method `GET` that returns an object with the form
+
+```
+{ apiStatus: true | false }
```

Comment:
I think the naming could be a little better. apiStatus `true / false` looks a little odd.

Reply:
What about just `status`?